x-pack/plugins/lens/public/visualizations/legacy_metric/metric_suggestions.ts

```diff
@@ -40,21 +40,19 @@ export function getSuggestions({
     return [];
   }
 
-  // do not return the legacy metric vis for the textbased mode (i.e. ES|QL)
```

Comment:
Also I still see the legacy metric in the switch and the suggestions

<img width="397" alt="image" src="https://github.com/elastic/kibana/assets/17003240/f05d3170-627f-4157-a7a2-c50b48a0ac93">

We agreed to hide it from both, right?

Reply:
Hidden! I thought it was hidden from suggestions, sorry! Now it's hidden from both.